Formula/auditbeat.rb

```diff
@@ -120,7 +113,7 @@ def plist
       touch testpath/"files/touch"
       sleep 30
       s = IO.readlines(testpath/"auditbeat/auditbeat").last(1)[0]
-      assert_match "\"action\":\[\"created\"\]", s
+      assert_match /"action":\["(initial_scan|created)"\]/, s
       realdirpath = File.realdirpath(testpath)
       assert_match "\"path\":\"#{realdirpath}/files/touch\"", s
     ensure
```

Comment:
This ensure isn't needed anymore. Brew kills all started processes

Reply:
Right, removed from all `*beat` formulae.